templates/ts-apollo-fullstack/client/src/generated-types.tsx

```diff
@@ -14,22 +14,22 @@ export type Scalars = {
 export type Comment = {
    __typename?: 'Comment',
   id: Scalars['ID'],
-  title: Scalars['String'],
```

Comment:
Was that generated ?

Reply:
The file was already there.And the queries for the comments wasn't working(Because it uses old comment model).So I had to change them.

Reply:
So this file is generated using graphql code generator. If we see value we can bring codegen for client side queries.